<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QDialog" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>288</width>
    <height>226</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0">
    <widget class="QLabel" name="labelPassword">
     <property name="text">
      <string>&amp;Password</string>
     </property>
     <property name="buddy">
      <cstring>lineEditPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="labelDatabase">
     <property name="text">
      <string>&amp;Database</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDatabase</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>56</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="3">
    <widget class="QCheckBox" name="chkBoxSave">
     <property name="toolTip">
      <string>The password will be saved locally on this computer. Please choose carefully.</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditPort">
     <property name="placeholderText">
      <string notr="true">5432</string>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QPushButton" name="pushButtonConnect">
     <property name="text">
      <string>Connect</string>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditDatabase"/>
   </item>
   <item row="4" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditPassword">
     <property name="echoMode">
      <enum>QLineEdit::EchoMode::PasswordEchoOnEdit</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Port</string>
     </property>
     <property name="buddy">
      <cstring>lineEditPort</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditUser"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="labelUser">
     <property name="text">
      <string>&amp;User</string>
     </property>
     <property name="buddy">
      <cstring>lineEditUser</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Host</string>
     </property>
     <property name="buddy">
      <cstring>lineEditHost</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditHost">
     <property name="placeholderText">
      <string notr="true">localhost </string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditHost</tabstop>
  <tabstop>lineEditPort</tabstop>
  <tabstop>lineEditUser</tabstop>
  <tabstop>lineEditPassword</tabstop>
  <tabstop>lineEditDatabase</tabstop>
  <tabstop>chkBoxSave</tabstop>
  <tabstop>pushButtonConnect</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
